.github/workflows/gradle-wrapper-validation.yml

```
@@ -9,4 +9,4 @@ jobs:
 
       - uses: actions/checkout@v4
 
-      - uses: gradle/wrapper-validation-action@v2
+      - uses: gradle/wrapper-validation-action@v3
```

Let's do the migration as well in one step:

```suggestion
      - uses: gradle/actions/wrapper-validation@v3
```